fe/fe-core/src/main/java/com/starrocks/qe/HDFSBackendSelector.java

```diff
@@ -205,53 +204,11 @@ private HashRing makeHashRing() {
                     new ComputeNodeFunnel(), nodes, kConsistenHashRingVirtualNumber);
         } else {
             hashRing = new RendezvousHashRing(Hashing.murmur3_128(), new TScanRangeLocationsFunnel(),
-                    new ComputeNodeFunnel(),
-                    nodes);
+                    new ComputeNodeFunnel(), nodes);
         }
         return hashRing;
     }
 
-    // Sort scan ranges, to shuffle them between hosts.
-    // Let's say sc1-h1, sc2-h1, sc3-h2, sc4-h2, then we will sort them as
-    // sc1-h1, sc3-h2, sc2-h1, sc4-h2, so h1, h2 will be interleaved assigned.
-    class ScanRagesSorter {
```

Comment:
Is there any case that this would help?

Reply:
No, just my imagination.  It could be a bad case,  but actually never met before in real case.